shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -928,12 +942,22 @@ - (void)replaceRangeLocal:(NSRange)range withText:(NSString*)text {
   [self setSelectedTextRangeLocal:[FlutterTextRange
                                       rangeWithNSRange:[self clampSelection:selectedRange
                                                                     forText:self.text]]];
+  _currentTextEditingDelta = [[FlutterTextEditingDelta alloc]
```

Comment:
Is this here just for updating the selection etc. with an equality delta?

Reply:
Yeah an equality delta is one where no text has changed, but the selection and/or composing regions have potentially been updated, so here we just set the current delta to an equality one for a selection update.

Reply:
I think you probably need to release the current `_currentTextEditingDelta` object before you assign a new one?